Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,702 @@
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: state
+                iscontext: true
+              right:
+                value:
+                  simple: active
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -650,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "3":
+    id: "3"
+    taskid: efeca58b-3c8f-40aa-8c2e-f9e84257b1bb
+    type: condition
+    task:
+      id: efeca58b-3c8f-40aa-8c2e-f9e84257b1bb
+      version: -1
+      name: Is there an endpoint to enrich?
+      description: Checks whether there is at least one endpoint to enrich (by hostname).
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "24"
+      - "1"
+    scriptarguments:
+      value:
+        simple: ${inputs.Hostname}
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: inputs.Hostname
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 215
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "4":
+    id: "4"
+    taskid: b95011d0-fd23-4e65-8ea9-560c5aead197
+    type: title
+    task:
+      id: b95011d0-fd23-4e65-8ea9-560c5aead197
+      version: -1
+      name: Done
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "5":
+    id: "5"
+    taskid: 44a06d71-1772-47af-8d41-8c6dbdd149e7
+    type: regular
+    task:
+      id: 44a06d71-1772-47af-8d41-8c6dbdd149e7
+      version: -1
+      name: Get host information from ePO
+      description: Gets host information from McAfee ePolicy Orchestrator.
+      script: epo|||epo-find-system
+      type: regular
+      iscommand: true
+      brand: epo
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      searchText:
+        complex:
+          root: inputs.Hostname
+          transformers:
+          - operator: uniq
+      verbose: {}
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -650,
+          "y": 996
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "8":
+    id: "8"
+    taskid: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+    type: condition
+    task:
+      id: 9fa921fa-d196-40ba-8419-ed0c4f838ab8
+      version: -1
+      name: Is Carbon Black Enterprise Response enabled?
+      description: Checks if there is any active instance of the Carbon Black Enterprise
```

Comment:
Checks if there is an active instance of the Carbon Black Enterprise Response integration enabled.